R/drive_upload.R

```diff
@@ -40,11 +41,33 @@
 #'
 #' ## clean-up
 #' drive_find("BioC_mirrors") %>% drive_rm()
+#'
+#' ## Upload a file with multiple parents
+#'
+#' ## Create mutltiple parent directories
+#' folder_1 <- drive_mkdir("folder_one")
+#' folder_2 <- drive_mkdir("folder_two")
+#'
+#' ## Upload the file into multiple parents. Notice here we are using
+#' ## the `addParents` parameter via the `...`.
+#' logo <- drive_upload(
+#'   R.home('doc/html/logo.jpg'),
+#'   folder_1,
+#'   addParents = folder_2$id
```

Comment:
FWIW I suspect you can both at the same time if you do `addParents = paste(folder_1$id, folder_2$id, sep = ",")`.

But, in any case, I'd prefer a less "out there" example. Such as adding a description? Or starring the file? I don't want to encourage anyone to do multiple parents!